packages/react-components/react-jsx-runtime/README.md

```diff
@@ -2,22 +2,28 @@
 
 **React JSX runtime for [Fluent UI React](https://react.fluentui.dev/)**
 
-[Fluent UI React](https://react.fluentui.dev/) requires the usage of a custom JSX runtime to support the [slots API](https://react.fluentui.dev/?path=/docs/concepts-developer-customizing-components-with-slots--page)
+[Fluent UI React](https://react.fluentui.dev/) requires the usage of a custom JSX runtime to support the [slots API](https://react.fluentui.dev/?path=/docs/concepts-developer-customizing-components-with-slots--docs)
 
 ## Usage
 
-This library should only be used in the case where you are trying to use the internal slot API of Fluent UI React. If you are not using the internal slot API, you should not need to use this library.
+> [!NOTE]
+> This custom JSX pragma should only be used in cases where you are trying to use the internal Fluent UI React **slot API in conjunction with `assertSlots()`**.
+>
+> If you are not using the internal slot API, don't use it.
 
-In case you want to re-compose a component and redeclare its render method then this API will be necessary, here's our documentation on [Rendering a component with slots](https://react.fluentui.dev/?path=/docs/concepts-developer-customizing-components-with-slots--page#rendering-components-with-slots)
+In case you want to re-compose a component and redeclare its render method then this API will be necessary, learn more on [Rendering a component with slots](https://react.fluentui.dev/?path=/docs/concepts-developer-customizing-components-with-slots--docs#rendering-components-with-slots)
```

Comment:
@bsunderhus  those linked SB docs doesn't describe anything related to "re-composing/declaring" besides  standard render "hook" implementation pattern. 

I'd rather update the wording here ( remove this completely ) or update the SB docs with examples of actual recomposition pattern. I prefer the latter.

thoughts?

Reply:
This entire documentation needs some 💖 with more examples of actual use cases. I agree that the latter would be ideal.

Reply:
kk I guess we have a follow up task then ! :D ty